charter/charter-detecting-unwanted-location-trackers.md

```diff
@@ -34,24 +34,25 @@ The WG protocol design will be guided by an intent to:
 
 The WG is expected to:
 
-1. Standardize a protocol between tracking accessories and nearby devices, which may:
+1. Standardize a protocol ("DULT protocol") between tracking accessories and nearby devices, which may:
 
  * Allow a tracking accessory to identify & advertise its presence when in a detectable mode
  * Allow a nearby device to trigger behavior on an unwanted tracking accessory to aid in determining its physical location
  * Allow nearby devices to fetch additional information about a tracker accessory
 
 
-2. Specify practices that accessory manufacturers can implement to deter malicious use of tracking accessories and support the implementation of the WG-specified protocol.
+2. Specify practices that accessory manufacturers can implement to deter malicious use of tracking accessories and support the implementation of the DULT protocol.
 
 
-3. Specify guidance for non-owner device platforms necessary to support implementation of the WG specified protocol
+3. Specify guidance for non-owner device platforms necessary to support implementation of the DULT protocol.
 
 
+4. Design mechanisms to ensure that devices that do not correctly implement or adhere to the DULT protocol can be detected and prevented from using crowdsourced location networks.
+
 
 The WG will not standardize an end-to-end platform-based unwanted tracking detection system or define requirements for interactions between accessory manufacturers and law enforcement. In addition, these items are out-of-scope:
 
- * Mechanisms for detecting tracking accessories that do not implement the protocol specified by the WG, and
- * Mechanisms for detecting whether a tracking accessory implements the protocol or allowing a tracking accessory to attest that it implements the protocol
+ * Mechanisms for detecting whether a tracking accessory implements the DULT protocol or allowing a tracking accessory to attest that it implements the protocol
```

Comment:
I don't understand why this is out of scope.

Reply:
I think it is in scope, just working on the right wording (and will update in another PR)